Solutions/AristaAwakeSecurity/Analytic Rules/HighMatchCountsByDevice.yaml

```diff
@@ -0,0 +1,64 @@
+id: 90b7ac11-dd6c-4ba1-a99b-737061873859
+name: Awake Security - High Match Counts By Device
+description: This query searches for devices with unexpectedly large number of activity match.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AristaAwakeSecurity
+    dataTypes:
+      - AwakeSecurity
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "Arista Networks" and DeviceProduct == "Awake Security"
+  | summarize Models=make_set(Activity), ASPMatchURLs=make_set(DeviceCustomString2), SourceIPs=make_set(SourceIP),
+    DestinationIPs=make_set(DestinationIP), ModelMatchCount=sum(EventCount), MaxSeverity=max(toint(LogSeverity)) by SourceHostName
+  | where ModelMatchCount > 1000 and MaxSeverity > 2
+  | extend SeverityName=iff(MaxSeverity == 0, "Informational", iff(MaxSeverity < 5, "Low", iff(MaxSeverity < 8, "Medium", "High")))
+queryFrequency: 1h
```

Comment:
The structure of the template should be fixed. queryFrequency, queryPeriod, triggerOperator, triggerThreshold, tactics, relevantTechniques should ideally come before the query.

Reply:
Fixed the structure for analytic rules